msg/CartesianPoint.msg

```diff
@@ -0,0 +1,3 @@
+geometry_msgs/Pose pose
+geometry_msgs/Twist velocity
+geometry_msgs/Accel acceleration
```

Comment:
Nit: need newline here?

Reply:
I don't think there's a consistent standard for this, at least not in this package :)